ui/src/containers/CreateVolume.js

```diff
@@ -384,6 +384,7 @@ const CreateVolume = props => {
                           type="button"
                           onClick={addLabel}
                           data-cy="add-volume-labels-button"
+                          outlined
```

Comment:
why `outlined`? because for me `outlined` button means `secondary` action and in this case `add volume` is a `primary` action.

Reply:
I just want to meet Use Design's proposal ;)

Reply:
![image](https://user-images.githubusercontent.com/47394132/70902449-18b62a80-1ffd-11ea-8139-ac669745730f.png)

Reply:
This is `add label` though, but I changed other buttons which should be the primary.